lib/fog/openstack/image_v2.rb

```diff
@@ -103,6 +103,8 @@ def credentials
               :openstack_management_url => @openstack_management_url }
           end
         end
+        class Upload # Exists for image_v2_upload_spec "describe"
```

Comment:
Please make the comment a proper rdoc comment (move the # blah above) or do not comment at all.